integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonDescribeFormattedCommand.scala

```diff
@@ -148,19 +148,24 @@ private[sql] case class CarbonDescribeFormattedCommand(
       ("## Compaction Information", "", ""),
       (CarbonCommonConstants.TABLE_MAJOR_COMPACTION_SIZE.toUpperCase,
         tblProps.getOrElse(CarbonCommonConstants.TABLE_MAJOR_COMPACTION_SIZE,
-        CarbonCommonConstants.DEFAULT_CARBON_MAJOR_COMPACTION_SIZE), ""),
+          CarbonProperties.getInstance().getProperty(CarbonCommonConstants.CARBON_MAJOR_COMPACTION_SIZE,
+            CarbonCommonConstants.DEFAULT_CARBON_MAJOR_COMPACTION_SIZE)), ""),
       (CarbonCommonConstants.TABLE_AUTO_LOAD_MERGE.toUpperCase,
         tblProps.getOrElse(CarbonCommonConstants.TABLE_AUTO_LOAD_MERGE,
-        CarbonCommonConstants.DEFAULT_ENABLE_AUTO_LOAD_MERGE), ""),
+          CarbonProperties.getInstance().getProperty(CarbonCommonConstants.ENABLE_AUTO_LOAD_MERGE,
+            CarbonCommonConstants.DEFAULT_ENABLE_AUTO_LOAD_MERGE)), ""),
       (CarbonCommonConstants.TABLE_COMPACTION_LEVEL_THRESHOLD.toUpperCase,
         tblProps.getOrElse(CarbonCommonConstants.TABLE_COMPACTION_LEVEL_THRESHOLD,
-        CarbonCommonConstants.DEFAULT_SEGMENT_LEVEL_THRESHOLD), ""),
+          CarbonProperties.getInstance().getProperty(CarbonCommonConstants.COMPACTION_SEGMENT_LEVEL_THRESHOLD,
```

Comment:
may be this line is long and exceed the threshold

Reply:
@qiuchenjian Yes...it is exceeding the threshold of 100 in a single line. Will modify it.